kuzzleio/kuzzle
# Receive the document updated content when calling document:update

## What does this PR do ?

Before this PR:
We could not retrieve the entire updated document in the request response.
Plus, Kuzzle was doing an extra request to ES in order to send notifications.

Now:
In Kuzzle, we passe the `source` option to ES when we update the document.
Doing that there is no more an extra request.
Finally, users can specify the option `source` in their request to receive the updated document in the request response.

Jira [Ticket](https://jira.kaliop.net/browse/KZL-1407) 





### How should this be manually tested?


`http://localhost:7512/index/test/document/_update?source=true`
see the updated document in `result._source`

`http://localhost:7512/index/test/document/_update`
do not send `result._source`


### Other changes



### Boyscout
asyncify methods
